[DOCS] Modifies schedule now transform API docs (#93519)
elastic/elasticsearch: docs/reference/transform/apis/schedule-now-transform.asciidoc

[role="xpack"]
[testenv="basic"]
[[schedule-now-transform]]
= Schedule now {transform} API

[subs="attributes"]
++++

> <titleabbrev>Shedule now {transform}</titleabbrev>

Reviewer: ```suggestion
<titleabbrev>Schedule now {transform}</titleabbrev>
```

[schedule-now-transform-desc]]
== {api-description-title}

> If you `_schedule_now` a {transform}, it will process the new data instantly,

Reviewer: IMO the use of the endpoint name as a verb is unwieldy. I think it's simpler like this:
```suggestion
When you run this API, the {transform} processes the new data instantly,
```

without waiting for the configured `frequency` interval.

> After the _schedule_now API is called, the transform will be processed again at

Reviewer: Likewise, I think this is simpler:
```suggestion
Subsequently, the transform will be processed again at
```
